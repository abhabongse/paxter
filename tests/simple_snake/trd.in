@!##{
def add_one(num):
    return num + 1
}##

Hello, my @strip{  full name   } is @name.
@tag[label="b"]{@name is @{age_last_year + 1} years old.}
@!{age_this_year = age_last_year + 1}

Do you know that 1 + 1 = @{1 + 1}?
