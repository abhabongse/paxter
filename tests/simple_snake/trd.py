name = "John Smith"
age_last_year = 47
strip = lambda token: token.strip()  # noqa: E731
tag = lambda token, label: f"<{label}>{token}</{label}>"  # noqa: E731
