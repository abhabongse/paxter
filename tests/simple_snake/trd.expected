

Hello, my full name is John Smith.
<b>John Smith is 48 years old.</b>


Do you know that 1 + 1 = 2?
